src/Symfony/Component/DependencyInjection/Tests/Loader/PhpFileLoaderTest.php

```diff
@@ -76,4 +76,17 @@ public function provideConfig()
             yield array('php7');
         }
     }
+
+    /**
+     * @expectedException \Symfony\Component\DependencyInjection\Exception\InvalidArgumentException
+     * @expectedExceptionMessage The service "child_service" cannot have a "parent" and also have "autoconfigure". Try disabling autoconfiguration for the service.
+     */
+    public function testAutoConfigureAndChildDefinitionNotAllowed()
```

Comment:
I've just added this one now, as it proves the issue with the missing use in  `AutoconfigureTrait`. But we should probably cover them all later.